Containers/Camera/Camera.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
+import { Camera, Permissions, FileSystem } from 'expo';
+
+export default class CameraExample extends Component {
+  constructor() {
+    super();
+    this.state = {
+      hasCameraPermission: null,
+      type: Camera.Constants.Type.back,
```

Comment:
This is setting which camera is being used front/back

Reply:
Oh cool